e2e-playwright/tests/nix-version.test.js

```diff
@@ -0,0 +1,16 @@
+// @ts-check
+const { test, expect } = require('@playwright/test');
+
+test('check nix version', async ({ page, request }) => {
+  await page.goto('/info');
+  const nixVersion = await page.locator(":text('Nix Version') + div").textContent();
+
+  const apiResponse = await request.get(`/api/data/nix-info`);
```

Comment:
We want to test that **some** version is shown in the browser, and not that the _correct_ version is displayed. Thus, there is no need to poll any API.